src-tauri/src/schema.rs

```diff
@@ -88,10 +97,46 @@ CREATE TABLE ratings (
 	date                 TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP
 ); 
 
+CREATE TABLE traits (
+    trait_id            INTEGER PRIMARY KEY,
+    name                TEXT NOT NULL UNIQUE,
+    range_low           TEXT NOT NULL DEFAULT '',
+    range_high          TEXT NOT NULL DEFAULT ''
+);
+
+CREATE TABLE items_to_traits ( 
+	item_id            INTEGER NOT NULL,
+	trait_id        INTEGER NOT NULL,
+    val             INTEGER NOT NULL,
```

Comment:
Can we change the `rating` to `val` as well instead of reusing the name 'rating'?